cmake/modules/thirdparty/SplitLibraryToCFlags.cmake

```diff
@@ -1,31 +0,0 @@
-
-function(split_library_to_cflags _lib _result)
-    if(_lib)
-        set(RESULT)
-        get_filename_component(_lib_name ${_lib} NAME_WE)
-        if(_lib_name STREQUAL "")
-        else()
-            string(REGEX REPLACE "^lib" "" _lib_name ${_lib_name})
```

Comment:
Before this PR, this was unsuitable for MSVC which doesn't use the `lib` prefix.